Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
To be fair, I've also seen `in` used as a variable name in multiple places, including in Base, but I definitely think it would look weird if it wasn't highlighted. (Though I think `new` is in a somewhat different category, and while it is a keyword I don't think visually there's much benefit in highlighting it.)